asf_core_data/pipeline/mcs/test/test_process_mcs_installations.py

```diff
@@ -21,42 +26,20 @@ def test_raw_columns_exist():
     Checks if raw columns are as expected.
     """
 
-    expected_cols = {
-        "Version Number",
-        "Commissioning Date",
-        "Address Line 1",
-        "Address Line 2",
-        "Address Line 3",
-        "County",
-        "Postcode",
-        "Local Authority",
-        "Total Installed Capacity",
-        "Green Deal Installation?",
-        "Installation Company Name",
-        "Products",
-        "Technology Type",
-        "Renewable System Design",
-        "Annual Space Heating Demand",
-        "Annual Water Heating Demand",
-        "Annual Space Heating Supplied",
-        "Annual Water Heating Supplied",
-        "Installation Requires Metering?",
-        "RHI Metering Status",
-        "RHI Metering Not Ready Reason",
-        "Number of MCS Certificates",
-        "Heating System Type",
-        "Fuel Type",
-        "Overall Cost",
-        "Installation Type",
-        "Installation Company MCS Number",
-    }
-    existing_cols = raw_installations.columns
+    expected_cols = base_config.historical_installations_rename_cols_dict.keys()
 
-    assert len(expected_cols.intersection(existing_cols)) == len(expected_cols)
+    existing_cols = raw_installations.columns
 
-    assert len([col for col in existing_cols if col not in expected_cols]) == 0
+    for col in expected_cols:
+        if col in ["Installation Type", " Installation Type"]:
+            assert ("Installation Type" in existing_cols) or (
+                " Installation Type" in existing_cols
+            )
+        else:
+            assert col in existing_cols
```

Comment:
Just wondering whether this could be done more easily using something like this, but I might be completely wrong:

```suggestion
    expected_cols = [col.strip() for col in expected_cols]
    existing_cols = [col.strip() for col in existing_cols]
    
    assert sorted(set(expected_cols)) == sorted(set(existing_cols))
```

If you only want to strip Installation type, you could do this:

```suggestion
    expected_cols = [col.strip()  if 'Installation Type' in col else col for col in expected_cols]
    existing_cols = [col.strip()  if 'Installation Type' in col else col for col in existing_cols]
    
    assert sorted(set(expected_cols)) == sorted(set(existing_cols))
```

Reply:
hey @athrado , thanks for the suggestion :) i'll use strip to avoid the if/else statement, but I'll keep the rest as is to compare variable by variable in order to see exactly where it breaks, when it breaks :)